### DISCUSSION TITLE
Share your achievement ideas

### DISCUSSION THREAD

**Discussion by devkennyy** [community/community#21237] [Other Feature Feedback, Questions, & Ideas]
If you have any ideas for the new achievements, I'd love to hear them (maybe Github will too 👀)

I'll go first:

**Hot Shot**
> Receive 10, 25, 50 upvotes in any discussion.

**Comment by AdamW1998** [comment]
You have shared a useless discussion point. Such a waste of time.

**Comment by NitinIsFit** [comment]
> If you have any ideas for the new achievements, I'd love to hear them (maybe Github will too 👀)
> 
> I'll go first:
> 
> **Hot Shot**
> > Receive 10, 25, 50 upvotes in any discussion. 

Sounds good 👍

**Reply by aayushchugh** [reply]
**Young Coder**
this should be given if you are under 16 years old

**Reply by FrenkyDema** [reply]
cool

**Comment by Acumane** [comment]
> Receive 10, 25, 50 upvotes in any discussion.

Ideally rewarded for upvoted/👍'ed replies on issue tickets as well
